src/index.ts

```diff
@@ -20,4 +21,42 @@ let bot = new Bot(
 	config.test_server
 );
 
+export const playOptions: PlayerOptions = {
+	leaveOnEnd: false,
+	leaveOnEmpty: false,
+	leaveOnStop: false,
+};
+
+export const player = new Player(bot.client);
```

Comment:
Could we move all of this into its own file so we aren't creating voice event handlers in index?